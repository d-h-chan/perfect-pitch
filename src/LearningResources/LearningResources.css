.learningResourcesHeader {
  border-left: 4px solid hotpink; 
}
