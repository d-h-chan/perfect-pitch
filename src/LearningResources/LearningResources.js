import React, { Component } from 'react';
import './LearningResources.css';

class LearningResources extends Component {
  render() {
    return (
      <>
        <h2 className="learningResourcesHeader">Learning Resources</h2>
        <ul>
          <li>
            <a href="https://www.figur8.net/2015/10/22/music-resources-for-perfect-pitch-training/">More Training Resources</a>
          </li>
          <li>
            <a href="https://newatlas.com/adults-perfect-pitch-training/37786/">Study on Perfect Pitch</a>
          </li>
        </ul>
      </>
    );
  }
}

export default LearningResources;