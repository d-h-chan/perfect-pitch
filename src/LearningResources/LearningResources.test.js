import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LearningResources from './LearningResources'

it('renders without errors', () => {
  const wrapper = shallow(<LearningResources />)
  expect(toJson(wrapper)).toMatchSnapshot()
})