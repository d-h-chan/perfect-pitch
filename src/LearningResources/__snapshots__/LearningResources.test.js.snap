// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without errors 1`] = `
<Fragment>
  <h2
    className="learningResourcesHeader"
  >
    Learning Resources
  </h2>
  <ul>
    <li>
      <a
        href="https://www.figur8.net/2015/10/22/music-resources-for-perfect-pitch-training/"
      >
        More Training Resources
      </a>
    </li>
    <li>
      <a
        href="https://newatlas.com/adults-perfect-pitch-training/37786/"
      >
        Study on Perfect Pitch
      </a>
    </li>
  </ul>
</Fragment>
`;
